### PR TITLE
Clarify how to override methods in C++/WinRT 

### PR DESCRIPTION
**Pull request by asklar**

### CODE REVIEW

**Comment by PRMerger5** [comment]
@asklar : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger12** [comment]
@asklar : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: windows-apps-src/cpp-and-winrt-apis/xaml-cust-ctrl.md [hunk 1]
```diff
@@ -203,7 +208,34 @@ struct BgLabelControl : BgLabelControlT<BgLabelControl>
...
+**C++**
+```cppwinrt
+namespace winrt::Example::implementation
+{
+    struct CustomVSM : CustomVSMT<CustomVSM>
+    {
+        CustomVSM() {}
+
+        bool GoToStateCore(winrt::Windows::UI::Xaml::Controls::Control const& control, winrt::Windows::UI::Xaml::FrameworkElement const& templateRoot, winrt::hstring const& stateName, winrt::Windows::UI::Xaml::VisualStateGroup const& group, winrt::Windows::UI::Xaml::VisualState const& state, bool useTransitions) {
+            return __super::GoToStateCore(control, templateRoot, stateName, group, state, useTransitions);
```
Comment (sylveon) [inline windows-apps-src/cpp-and-winrt-apis/xaml-cust-ctrl.md]: Maybe remove the use of the `__super` compiler extension here - cppwinrt provides `base_type` serving the same purpose
```suggestion
            return base_type::GoToStateCore(control, templateRoot, stateName, group, state, useTransitions);
```

Comment (asklar) [reply windows-apps-src/cpp-and-winrt-apis/xaml-cust-ctrl.md]: good idea, thanks

**Comment by PRMerger9** [comment]
@asklar : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by sylveon — APPROVED** [review]

**Comment by ktoliver** [comment]
#label:"aq-pr-triaged"

**Comment by PRMerger4** [comment]
@stevewhims : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger15** [comment]
@stevewhims : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by stevewhims — APPROVED** [review]

**Pull request merged**